image/templates/helm/stackrox-secured-cluster/templates/sensor.yaml.htpl

```diff
@@ -110,6 +110,8 @@ spec:
         {{- if ._rox.env.openshift }}
         - name: ROX_OPENSHIFT_API
           value: "true"
+        - name: ROX_NODE_INDEX_ENABLED
+          value: "true"
```

Comment:
Could you comment on why this was needed in this PR?
Do we need to do some other changes to the charts before we release?

Reply:
I was reminded in the last sync meeting with Scanner team that there is a possibility that customers want to switch Node Indexing separately from Scanner v4. I plan to decouple them in a follow-up PR.
I want to make sure that Node Indexing is primarily switched on or off by setting this var, not by the state of the Scanner v4 feature flag.